Jenkinsfile

```diff
@@ -149,6 +149,13 @@ jobWrappers {
       stage('Checkout') {
         checkout scm
       }
+      stage('Prechecks') {
+        def findbugsProjects = [
+            ":hive-shims-aggregator",
+            ":hive-shims-common"
+        ]
+        buildHive("-Pfindbugs -pl " + findbugsProjects.join(",") + " -am compile findbugs:check")
```

Comment:
I think at some point we can put this after the main "compile" step - so that it doesn't need "compile"